Jenkinsfile

```
@@ -0,0 +1,101 @@
+#!groovy
+@Library('github.com/cloudogu/ces-build-lib@1.48.0')
+import com.cloudogu.ces.cesbuildlib.*
+
+node('docker') {
+
+    def git = new Git(this)
+    def mvn = new MavenWrapperInDocker(this, 'azul/zulu-openjdk-alpine:11.0.10')
+
+    timestamps {
+        catchError {
+            timeout(activity: false, time: 60, unit: 'MINUTES') {
+
+
+                stage('Checkout') {
+                    git branch: 'main', url: 'https://github.com/cloudogu/gitops-playground'
+                    dir('etcd') {
+                        checkout scm
+                    }
+                }
+
+                yamllintImage = "cytopia/kubeval:0.15"
```

should be `kubevalImage`

```suggestion
                kubevalImage = "cytopia/kubeval:0.15"
```